app/controllers/project/donate.js

```diff
@@ -54,15 +56,19 @@ export default Controller.extend({
     }
   },
 
-  // credit card token
+  // Stripe token
 
-  _createCreditCardToken(cardParams) {
-    let stripeCard = this._tokenParams(cardParams);
-    let stripe = get(this, 'stripe');
+  async _createStripeToken(stripeElement) {
+    let stripe = get(this, 'stripev3');
 
-    return stripe.card.createToken(stripeCard)
-                      .then((stripeResponse) => RSVP.resolve(stripeResponse))
-                      .catch((reason) => this._handleCreditCardTokenError(reason));
+    return await stripe.createToken(stripeElement).then(({ error, token }) => {
+      if (error) {
+        return this._handleCreditCardTokenError(error);
+      }
+      if (token) {
```

Comment:
Realistically, the case should probably not happen ever, but I still seems more correct to have an `else if` here.